apps/condo/domains/common/hooks/useValidations.ts

```diff
@@ -0,0 +1,64 @@
+import { useIntl } from '@core/next/intl'
+import { Rule } from 'rc-field-form/lib/interface'
+const { normalizePhone } = require('@condo/domains/common/utils/phone')
+
+type CombinerParams = Array<Rule | Array<Rule>>
+
+type ValidatorHookReturnType = {
+    combiner: (...rules: CombinerParams) => Rule[]
+    messageChanger: (rule: Rule, message: string) => Rule
+    requiredValidator: Rule
+    phoneValidator: Rule
+    emailValidator: Rule
+    trimValidator: Rule
+}
+
+const combiner = (...rules: CombinerParams) => {
```

Comment:
combine

Reply:
Why should we introduce this combiner function if as the result we will get an array of validators? As i can imagine we can use validators like: 

`
const { combiner, messageChanger, phoneValidator, emailValidator, requiredValidator } = useValidations()
const validators = {
    phone: [requiredValidator, phoneValidator, alreadyRegisteredPhoneValidator],
}
`

Reply:
Idea was that common rules can be "Rule" or "Rule[]", so, combine was good idea in my head, it will make you not think about lists deconstruction...so, should we left it?

Reply:
resolved 👍